generator/src/test/java/com/scottlogic/deg/generator/cucumber/SelectiveViolation.feature

```diff
@@ -68,7 +68,7 @@ Background:
       | 10        |
       | null      |
 
-  Scenario: Running a 'containingRegex' request for a single character (a) should be successfulljkljkljkjkl7
+  Scenario: Running a 'containingRegex' request for a single character (a) should be successful
```

Comment:
Unusual change, can we check there are no duplicate names now?

Reply:
your right, there is already a test in containingregex with this name, I'll revert the change